api_server/src/main/java/io/zetch/app/domain/User.java

```diff
@@ -39,4 +52,13 @@ public String getUsername() {
     public void setUsername(String username) {
         this.username = username;
     }
+
+    @Override
+    public String toString() {
+        return "User{" +
+                "username='" + username + '\'' +
+                ", name='" + name + '\'' +
+                ", email='" + email + '\'' +
+                '}';
+    }
 }
```

Comment:
The reason the red stop mark is here is that the newline after the end of file is recommended. It will be automatically corrected if we will setup our linter!